src/metatrain/experimental/nanopet/modules/encoder.py

```diff
@@ -0,0 +1,57 @@
+from typing import Dict
+
+import torch
+
+
+class Encoder(torch.nn.Module):
```

Comment:
could this have a docstring explaining what this encoder is (looks like a MLP), and why?